gammapy/modeling/parameter.py

```diff
@@ -427,7 +455,10 @@ def to_dict(self):
 
         if self._link_label_io is not None:
             output["link"] = self._link_label_io
-
+        if self.prior is not None:
+            output["prior"] = self.prior.to_dict()["prior"]
+        else:
+            output["prior"] = None
```

Comment:
Maybe you don't need those two lines. If no prior is defined it does not need to be explicit.